internal/executor/executor.go

```diff
@@ -0,0 +1,109 @@
+// Package executor implements the replicant transaction execution service
+package executor
+
+import (
+	"context"
+	"net/http"
+	"time"
+
+	"github.com/Unbabel/replicant/driver"
+	gd "github.com/Unbabel/replicant/driver/go"
```

Comment:
The name of the package (since it was changed) should be more explicit something like `godriver`.